### PR TITLE
Canbus & Common : start merge common configs

### PR DESCRIPTION
**Pull request by Capri2014**

### CODE REVIEW

File: modules/canbus/vehicle/vehicle_controller.h [hunk 1]
```diff
@@ -152,6 +153,7 @@ class VehicleController {
 
  protected:
   canbus::VehicleParameter params_;
```
Comment (startcode) [inline modules/canbus/vehicle/vehicle_controller.h]: delete params_?

File: modules/canbus/vehicle/gem/gem_controller.cc [hunk 2]
```diff
@@ -408,8 +409,7 @@ void GemController::Steer(double angle) {
     return;
   }
   // convert to radians
-  const double real_angle =
-      params_.max_steer_angle() * angle / 100.0 * M_PI / 180.0;
+  const double real_angle = vehicle_params_.max_steer_angle() * angle / 100.0;
```
Comment (startcode) [inline modules/canbus/vehicle/gem/gem_controller.cc]: the comment "conver to radians" does not match with the code.

File: modules/canbus/vehicle/vehicle_controller.h (hunk 1 above)
Comment (Capri2014) [reply modules/canbus/vehicle/vehicle_controller.h]: @startcode It is still half-done,  so I left the legacy here for now

File: modules/canbus/vehicle/gem/gem_controller.cc (hunk 2 above)
Comment (Capri2014) [reply modules/canbus/vehicle/gem/gem_controller.cc]: @startcode Will changle

Comment (Capri2014) [reply modules/canbus/vehicle/gem/gem_controller.cc]: @startcode Done

**Pull request merged**